example/contract-schema.ts

```diff
@@ -5,22 +5,24 @@ import { InMemorySigner } from '@taquito/signer';
 async function example() {
     const provider = 'https://hangzhounet.api.tez.ie';
     const signer: any = new InMemorySigner('edskRtmEwZxRzwd1obV9pJzAoLoxXFWTSHbgqpDBRHx1Ktzo5yVuJ37e2R4nzjLnNbxFU4UiBU1iHzAy52pK5YBRpaFwLbByca');
-    const tezos = new TezosToolkit(provider);
+        const tezos = new TezosToolkit(provider);
     tezos.setSignerProvider( signer );
     try {
-        const contract = await tezos.contract.at('KT1KxMdSBa9ippgTWwU7d1FkzTF5CirUkcp4');
+        console.log("signer pkh:");
+        console.log(await signer.publicKeyHash());
+        const contract = await tezos.contract.at('KT1SyjphGj51CadmmbL8dmKwMA4VyMQdtkjB');
         console.log("Printing contract methods...");
         console.log(contract.methods);
         console.log("Showing initial storage...");
         console.log(await contract.storage())
-        const op = await contract.methods.mint("tz2GFmGv5iWy4wYLPxJ3BbL377t2QsLPCGp8", 100).send()
+        const op = await contract.methods.mint("tz1bwsEWCwSEXdRvnJxvegQZKeX5dj6oKEys", 100).send()
         console.log('Awaiting confirmation...');
         await op.confirmation();
         console.log(op.hash, op.includedInBlock);
         console.log("Showing final storage...");
         console.log(await contract.storage())
     } catch (ex) {
-        console.log(ex)
+        console.log(ex.errors[1].with)
```

Comment:
not all errors will match ".errors[1].with"

Reply:
Fixed. (restored to original)